prow/jobs_config/knative/serving.yaml

```diff
@@ -148,7 +148,7 @@ jobs:
     - --run-test
     - ./test/e2e-external-domain-tls-tests.sh --contour-version latest --run-http01-external-domain-tls-tests
 
-  - name: gateway-api-latest
+  - name: gateway-api-istio
```

Comment:
@upodroid do you know if there's a way to preserve the history of the old job name in test-grid?

Reply:
I don't think so.